### DISCUSSION TITLE
Glyph Icon

### DISCUSSION THREAD

**Discussion by taooceros** [Flow-Launcher/Flow.Launcher#592] [Ideas]
We can add a Property for storing Glyph for results, and display it as an alternative icon other than the specific image.
At the same time, probably we can allow plugins to modify the specific font family of specific result.

It is also possible that people would like to use Glyph Icon instead of an Icon Image.

https://github.com/Flow-Launcher/Flow.Plugin.WindowsSettings/pull/2

**Comment by deefrawley** [comment]
If it gives plugin devs the ability to use things like [Nerd Fonts](https://www.nerdfonts.com/) in search results then it might open up some interesting possibilities.

**Reply by taooceros** [reply]
That may requires extra font installed, but it will enable users to use Windows builtin Asset font (I don't remember the font name 😂)

**Reply by pc223** [reply]
No way for plugins kind of dynamically add fonts in? 💭

**Reply by taooceros** [reply]
> No way for plugins kind of dynamically add fonts in? 💭 

I think users may still need to install it manually? But not sure, probably there's a way to embed a font.

**Reply by jjw24** [reply]
If it's a simple non-interactive font install, we can do it the way plugins install their app/service dependency

**Reply by deefrawley** [reply]
Can't apps bundle and directly reference TTF files without Windows installing them into System32/Fonts? If so then the file just becomes an artefact of the plugin

**Comment by pc223** [comment]
How hard is this gonna be? If it takes too much time I think for now we can try to convert the glyph to images for that plugin.

**Reply by taooceros** [reply]
Not really hard. Just needs to check the arrangement.

**Comment by taooceros** [comment]
Probably we can let plugin results choose either glyph or icon
